Multiplatform/Shared/AppDefaults.swift

```diff
@@ -134,7 +134,9 @@ final class AppDefaults: ObservableObject {
 		}
 		set {
 			AppDefaults.store.set(newValue.rawValue, forKey: Key.userInterfaceColorPalette)
-			objectWillChange.send()
+			DispatchQueue.main.asyncAfter(deadline: .now() + 0.5, execute: {
```

Comment:
This delay makes the transition between modes a little more natural, especially on iOS, and it happens just after the navigation transition.